src/main/java/uk/gov/hmcts/divorce/caseworker/service/task/SendAosPackToRespondent.java

```diff
@@ -22,7 +24,8 @@ public CaseDetails<CaseData, State> apply(final CaseDetails<CaseData, State> cas
         final Long caseId = caseDetails.getId();
         final CaseData caseData = caseDetails.getData();
 
-        if (caseData.getApplication().isCourtServiceMethod()) {
+        if (caseData.getApplication().isCourtServiceMethod()
+                && NO.equals(caseData.getApplication().getApplicant1WantsToHavePapersServedAnotherWay())) {
```

Comment:
May be we can use 

`!YES.equals(caseData.getApplication().getApplicant1WantsToHavePapersServedAnotherWay())`

So that along with NO, this condition will be true for NULL as well

Reply:
I actually had it as you have suggested to counter the null values and discussed with Nitin the same this morning but not sure why it has reverted to `NO.equals...` 😆 i will update this now

Reply:
Changed from `NO.equals..` to `!YES.equals...` for null check as well